src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -143,9 +143,28 @@ public static class TimeSeriesCatalog
         /// </format>
         /// </example>
         public static SrCnnAnomalyEstimator DetectAnomalyBySrCnn(this TransformsCatalog catalog, string outputColumnName, string inputColumnName,
-            int windowSize=64, int backAddWindowSize=5, int lookaheadWindowSize=5, int averageingWindowSize=3, int judgementWindowSize=21, double threshold=0.3)
+            int windowSize = 64, int backAddWindowSize = 5, int lookaheadWindowSize = 5, int averageingWindowSize = 3, int judgementWindowSize = 21, double threshold = 0.3)
             => new SrCnnAnomalyEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, windowSize, backAddWindowSize, lookaheadWindowSize, averageingWindowSize, judgementWindowSize, threshold, inputColumnName);
 
+        /// <summary>
+        /// Create <see cref="DTRootCauseLocalizationEstimator"/>, which localizes root causes using decision tree algorithm.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// The column data is an instance of <see cref="Microsoft.ML.TimeSeries.RootCause"/>.</param>
+        /// <param name="inputColumnName">Name of the input column.
+        ///  The column data is an instance of <see cref="Microsoft.ML.TimeSeries.RootCauseLocalizationInput"/>.</param>
+        /// <param name="beta">The weight parameter in score. The range of the parameter should be in [0,1].</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[LocalizeRootCauseByDT](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/LocalizeRootCauseByDT.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public static DTRootCauseLocalizationEstimator LocalizeRootCauseByDT(this TransformsCatalog catalog, string outputColumnName, string inputColumnName = null, double beta = 0.5)
```

Comment:
A few suggestions about naming:
- DT is not very descriptive
- Calling it DecisionTreeRootCauseLocalizationEstimator is perhaps a bit too long
- Both the above hard code the algorithm used for root cause estimation

Would it be better to just call it RootCauseLocalizationEstimator (and the function to be named LocalizeRootCause or EstimateRootCause) so that way we can have a shorter name also remain open to adding other algorithms in the future if needed?

Reply:
> A few suggestions about naming:
> 
> * DT is not very descriptive
> * Calling it DecisionTreeRootCauseLocalizationEstimator is perhaps a bit too long
> * Both the above hard code the algorithm used for root cause estimation
> 
> Would it be better to just call it RootCauseLocalizationEstimator (and the function to be named LocalizeRootCause or EstimateRootCause) so that way we can have a shorter name also remain open to adding other algorithms in the future if needed?

Agree with you, have upated